internal/cmd/dev.go

```diff
@@ -45,6 +52,9 @@ var devCmd = &cobra.Command{
 			}
 		}
 
+		fmt.Printf("Using sqld version: ")
+		sqldVersion()
```

Comment:
Why is the version printed twice? Once only when the flag is set and once no matter what the flag value is.

Reply:
Before logging the `sqld` is running at the port, I wanted to show the version used. It's OK not to include there.